singer/metadata.py

```diff
@@ -21,3 +21,26 @@ def write(compiled_metadata, breadcrumb, k, val):
 
 def get(compiled_metadata, breadcrumb, k):
     return compiled_metadata.get(breadcrumb, {}).get(k)
+
+def get_standard_metadata(schema=None, schema_name=None, key_properties=None,
```

Comment:
Shouldn't this just take a catalog?

Reply:
It could take a catalog entry, but we'd have to change our "pattern" of creating the catalog entry and the metadata at the same time

Reply:
Yeah. The goal of this isn't to be a one stop shop for metadata, but to give a better API for creating the default metadata. The catalog input could be part of higher-order catalog functions.